controllers/user/api/v1/user_types.go

```diff
@@ -52,7 +52,7 @@ const (
 // UserStatus defines the observed state of User
 type UserStatus struct {
 	// Phase is the recently observed lifecycle phase of user
-	//+kubebuilder:default:=UserUnknown
+	//+kubebuilder:default:=Unknown
```

Comment:
Phase means what?

Reply:
sync usergroup kubeconfig usergroupbinding reousource final status.  default is unknow , controller running modify it.